scanner/image/scanner/konflux.Dockerfile

```
@@ -0,0 +1,71 @@
+ARG MAPPINGS_REGISTRY=registry.access.redhat.com
+ARG MAPPINGS_BASE_IMAGE=ubi8
+ARG MAPPINGS_BASE_TAG=latest
+ARG BASE_REGISTRY=registry.access.redhat.com
+ARG BASE_IMAGE=ubi8-minimal
+ARG BASE_TAG=latest
+
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_8_1.20 as builder
+
+ENV CGO_ENABLED=1
+ENV GOFLAGS=""
+ENV GOTAGS="release"
+ENV RELEASE_TAG="$CI_VERSION"
+ENV CI=1
+
+COPY . /src
+WORKDIR /src
+
+RUN scripts/konflux/fail-build-if-git-is-dirty.sh
+
+RUN make -C scanner NODEPS=1 CGO_ENABLED=${CGO_ENABLED} image/scanner/bin/scanner copy-scripts
+
+FROM ${BASE_REGISTRY}/${BASE_IMAGE}:${BASE_TAG}
+
+LABEL \
+    com.redhat.component="rhacs-scanner-container" \
+    com.redhat.license_terms="https://www.redhat.com/agreements" \
+    description="This image supports image scanning for RHACS" \
+    io.k8s.description="This image supports image scanning for RHACS" \
+    io.k8s.display-name="scanner" \
+    io.openshift.tags="rhacs,scanner,stackrox" \
+    maintainer="Red Hat, Inc." \
+    name="rhacs-scanner-rhel8" \
+    source-location="https://github.com/stackrox/scanner" \
```

```suggestion
    com.redhat.component="rhacs-scanner-v4-container" \
    com.redhat.license_terms="https://www.redhat.com/agreements" \
    description="This image supports image scanning for RHACS" \
    io.k8s.description="This image supports image scanning for RHACS" \
    io.k8s.display-name="scanner-v4" \
    io.openshift.tags="rhacs,scanner,stackrox" \
    maintainer="Red Hat, Inc." \
    name="rhacs-scanner-v4-rhel8" \
    source-location="https://github.com/stackrox/stackrox" \
```

I also wonder if we should mention `v4` in `summary` and `*description`.

---

I added something about v4 to summary and description